x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations_settings/curations_settings.tsx

```diff
@@ -0,0 +1,166 @@
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiButtonEmpty, EuiCallOut, EuiSpacer, EuiSwitch, EuiText, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { FormattedMessage } from '@kbn/i18n/react';
+
+import { docLinks } from '../../../../../shared/doc_links';
+import { LicensingLogic } from '../../../../../shared/licensing';
+import { Loading } from '../../../../../shared/loading';
+import { EuiButtonTo } from '../../../../../shared/react_router_helpers';
+import { SETTINGS_PATH } from '../../../../routes';
+import { DataPanel } from '../../../data_panel';
+import { LogRetentionLogic, LogRetentionOptions } from '../../../log_retention';
+
+import { CurationsSettingsLogic } from './curations_settings_logic';
+
+export const CurationsSettings: React.FC = () => {
+  const { hasPlatinumLicense } = useValues(LicensingLogic);
+
+  const {
+    curationsSettings: { enabled, mode },
+    dataLoading,
+  } = useValues(CurationsSettingsLogic);
+  const { loadCurationsSettings, toggleCurationsEnabled, toggleCurationsMode } =
+    useActions(CurationsSettingsLogic);
+
+  const { isLogRetentionUpdating, logRetention } = useValues(LogRetentionLogic);
+  const { fetchLogRetention } = useActions(LogRetentionLogic);
+
+  const analyticsDisabled = !logRetention?.[LogRetentionOptions.Analytics].enabled;
+
+  useEffect(() => {
+    loadCurationsSettings();
+    fetchLogRetention();
+  }, []);
+
+  if (!hasPlatinumLicense)
+    return (
+      <DataPanel
+        title={
+          <h2>
+            {i18n.translate(
+              'xpack.enterpriseSearch.appSearch.curations.settings.licenseUpgradeCTATitle',
+              {
+                defaultMessage: 'Introducing Automated Curations',
+              }
+            )}
+          </h2>
+        }
+        subtitle={
+          <FormattedMessage
+            id="xpack.enterpriseSearch.appSearch.curations.settings.licenseUpgradeCTASubtitle"
+            defaultMessage="Upgrade to a {platinumLicenseName} license to harness the power of machine learning. By analyzing your engine's analytics, App Seearch is able to suggest new or updated curations. Effortlessly help your users find exactly what they're looking for. Start a free trial today."
+            values={{
+              platinumLicenseName: <strong>Platinum</strong>,
+            }}
+          />
+        }
+        action={
+          <EuiButtonTo to="/app/management/stack/license_management" shouldNotCreateHref>
+            {i18n.translate('xpack.enterpriseSearch.start30DayTrialButtonLabel', {
+              defaultMessage: 'Start a 30-day trial',
+            })}
+          </EuiButtonTo>
+        }
+      >
+        <EuiButtonEmpty
+          target="_blank"
+          iconType="popout"
+          href={`${docLinks.enterpriseSearchBase}/license-management.html`}
+        >
+          {i18n.translate('xpack.enterpriseSearch.licenseUpgradeLink', {
```

Comment:
Should we nest this under `xpack.enterpriseSearch.appSearch.curations.settings.`?

Reply:
I've done this in dc7abee